src/Slider.jsx

```diff
@@ -8,11 +8,21 @@ import * as SliderConstants from './constants/SliderConstants';
 import linear from './algorithms/linear';
 
 function getClassName(props) {
-  const orientation = props.orientation === 'vertical'
+  let classNames = ['rheostat'];
+
+  classNames.push(props.orientation === 'vertical'
     ? 'rheostat-vertical'
-    : 'rheostat-horizontal';
+    : 'rheostat-horizontal');
```

Comment:
the `);` needs to be on a line by itself; perhaps a cleaner option is `classNames.push(orientation)`, or `const classNames = ['rheostat', orientation]` after the ternary?

Reply:
fixed